pkgs/applications/graphics/oculante/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, cmake
+, pkg-config
+, openssl
+, fontconfig
+, nasm
+, netcat
+, libX11
+, libXcursor
+, libXrandr
+, libXi
+, libGL
+, stdenv
+, libobjc
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "oculante";
+  version = "0.6.35";
+
+  src = fetchFromGitHub {
+    owner = "woelper";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-I6s6hDYu2y9YCfn9smoijZ7c+dNhuQDSU415fDR0SmM=";
+  };
+
+  cargoSha256 = "sha256-N8HjKvz0cvY9VVvMGfUzxYV7rfWXnHTcm6VlCVQ4Z2M=";
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  checkFlagsArray = [ "--skip=tests::net" ]; # requires network access
+
+  buildInputs =
+    [
+      openssl
+      fontconfig
+      nasm
+    ] ++
+    lib.optionals stdenv.isLinux [
+      libX11
+      libXcursor
+      libXi
+      libXrandr
+      libGL
+    ] ++
+    lib.optionals stdenv.isDarwin [
+      libobjc
+    ];
+
+  postFixup = lib.optionalString stdenv.isLinux ''
+    patchelf $out/bin/oculante \
+      --add-rpath ${lib.makeLibraryPath [libGL]}
+  '';
+
+  meta = with lib; {
+    description = "A minimalistic crossplatform image viewer written in Rust";
+    homepage = "https://github.com/woelper/oculante";
+    changelog = "https://github.com/woelper/oculante/blob/${version}/CHANGELOG.md";
+    license = licenses.mit;
+    maintainers = with maintainers; [ dit7ya ];
```

Building on darwin seems non-trivial, I think we can just mark it broken for now
```suggestion
    maintainers = with maintainers; [ dit7ya ];
    broken = stdenv.isDarwin;
```